### PR TITLE
Clarify origin of ID parameter for path filter creation

### PR DESCRIPTION
**Pull request by raskchanky**

### CODE REVIEW

File: website/content/api-docs/system/replication/replication-performance.mdx [hunk 1]
```diff
@@ -266,7 +266,9 @@ selected secondary. In deny mode, the mount and namespace paths are excluded.
 
 ### Parameters
 
-- `id` `(string: <required>)` – Specifies the unique performance secondary identifier.
+- `id` `(string: <required>)` – Specifies the unique performance secondary identifier. Note that this identifier is
+  not arbitrary. This should be the secondary cluster ID as reported by the `cluster_id` field of the [replication status](/vault/api-docs/system/replication#check-status) API
+  endpoint. If this ID ever changes, this endpoint will need to be run again with the new ID.
```
[the remark this answers is not in the document]
Comment (ncabatoff) [reply website/content/api-docs/system/replication/replication-performance.mdx:271]: Only case I can think of for PR is update-primary (old style).  For DR, a promotion/demotion followed by reconfiguring the former primary as a secondary will I think entail a new secID, but that's irrelevant for path filters.

[the remark this answers is not in the document]
Comment (ncabatoff) [reply website/content/api-docs/system/replication/replication-performance.mdx:271]: Maybe the way to phrase it is less as "if the ID changes" and more like: every time a perf secondary token is created, a new secondary ID must be used.  So if you're using update-primary with a new perf secondary token, that should tell you that you need to redo your path filters.

[the remark this answers is not in the document]
Comment (akshya96) [reply website/content/api-docs/system/replication/replication-performance.mdx:271]: I was looking at our code yesterday and it looks like this happens when a secondary token is created. I agree with Nick's latest comment.

[the remark this answers is not in the document]
Comment (raskchanky) [reply website/content/api-docs/system/replication/replication-performance.mdx]: I updated the description some. What are y'alls thoughts on the new text?

**Review by ncabatoff — APPROVED** [review]

**Pull request merged**